_includes/navbar.html

```diff
@@ -66,16 +66,21 @@
             <li{% if page.url contains '/downloads.html' %} class="active"{% endif %}><a href="{{ baseurl_i18n }}/downloads.html">{{ site.data.i18n[page.language].downloads }}</a></li>
 
             <!-- Getting Started -->
-            <li>
-              <a href="{{ site.docs-stable }}/{% if page.language != 'en' %}{{ page.language }}/{% endif %}getting-started/index.html" target="_blank">{{ site.data.i18n[page.language].getting_started }} <small><span class="glyphicon glyphicon-new-window"></span></small></a>
+            <li class="dropdown">
+              <a class="dropdown-toggle" data-toggle="dropdown" href="#">{{ site.data.i18n[page.language].getting_started }}<span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a href="{{ site.docs-stable }}/{% if page.language != 'en' %}{{ page.language }}/{% endif %}getting-started/index.html" target="_blank">With Flink <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
+                <li><a href="https://ci.apache.org/projects/flink/flink-statefun-docs-stable/getting-started/project-setup.html" target="_blank">With Flink Stateful Functions <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
+              </ul>
             </li>
 
             <!-- Documentation -->
             <li class="dropdown">
               <a class="dropdown-toggle" data-toggle="dropdown" href="#">{{ site.data.i18n[page.language].documentation }}<span class="caret"></span></a>
               <ul class="dropdown-menu">
-                <li><a href="{{ site.docs-stable }}" target="_blank">{{site.FLINK_VERSION_STABLE_SHORT}} (Latest stable release) <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
-                <li><a href="{{ site.docs-snapshot }}" target="_blank">Master (Latest Snapshot) <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
+                <li><a href="{{ site.docs-stable }}" target="_blank">Flink {{site.FLINK_VERSION_STABLE_SHORT}} (Latest stable release) <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
+                <li><a href="{{ site.docs-snapshot }}" target="_blank">Flink Master (Latest Snapshot) <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
+                <li><a href="https://ci.apache.org/projects/flink/flink-statefun-docs-stable/" target="_blank">Flink Stateful Functions (Latest stable release) <small><span class="glyphicon glyphicon-new-window"></span></small></a></li>
```

Comment:
I think we can add a `docs-statefun-stable` and `docs-statefun-snapshot` configs to the `_config.yml` file for this.
I'll address this while merging.